clients/reality-data/src/RealityDataClient.ts

```diff
@@ -374,10 +367,9 @@ export class DataLocation extends WsgInstance {
  * Client wrapper to Reality Data Service.
 // SWB What to do here, keep PW reference?
  * An instance of this class is used to extract reality data from the ProjectWise Context Share (Reality Data Service)
-  // SWB
- * Most important methods enable to obtain a specific reality data, fetch all reality data associated to a project and
-  // SWB
- * all reality data of a project within a provided spatial extent.
+ * Most important methods enable to obtain a specific reality data, fetch all reality data associated with an iTwin and
+// SWB Is this referring to iTwins or "project extents" which currently are keeping their name?
```

Comment:
I think the changes you made to this comment make sense. Lets leave what you have here.

Reply:
Sounds good